client/landing/stepper/declarative-flow/internals/steps-repository/podcast-title/index.tsx

```diff
@@ -27,16 +27,25 @@ const PodcastTitleStep: Step = function PodcastTitleStep( { navigation } ) {
 	const isLookingUpMatchingAnchorSites = useDetectMatchingAnchorSite();
 
 	const PodcastTitleForm: React.FC = () => {
-		//Sets the site title from the API on first load if a custom title has not been set
-		useSiteTitle();
+		//Get the podcast title from the API
+		const podcastTitle = usePodcastTitle();
 		const { siteTitle } = useSelect( ( select ) => select( ONBOARD_STORE ).getState() );
-		const [ formTouched, setFormTouched ] = useState( false );
-
+		const hasSiteTitle = siteTitle.length > 0;
 		const { setSiteTitle, setAnchorPodcastId, setAnchorEpisodeId, setAnchorSpotifyUrl } =
 			useDispatch( ONBOARD_STORE );
 		const { anchorFmPodcastId, isAnchorFmPodcastIdError, anchorFmEpisodeId, anchorFmSpotifyUrl } =
 			useAnchorFmParams();
 
+		//If we don't have a custom title in the store, use the podcast title from the API
+		useEffect( () => {
+			if ( podcastTitle && ! hasSiteTitle ) {
```

Comment:
I believe this condition is causing a small problem. If you try to delete the whole title, on the last letter the text gets reset to the default title.
![2022-05-10 09 41 04](https://user-images.githubusercontent.com/1234758/167630742-ff705f47-bb9f-466e-b084-64e69c5d4cec.gif)

Reply:
Ooh neat bug! I'll take a look at this.